packages/regl-worldview/src/camera/CameraStore.js

```diff
@@ -56,47 +55,50 @@ export default class CameraStore {
     this.state = state;
   };
 
-  _setStateAndNotify(cameraState: CameraState) {
-    // we only want to call onChange when cameraState values actually changes, as each
-    // onChange will trigger Worldview to rerender by forceUpdate or onCameraStateChange
-    if (!isEqual(this.state, cameraState)) {
-      this.state = cameraState;
-      this._onChange(this.state);
-    }
-  }
-
   cameraRotate = ([x, y]: Vec2) => {
+    if (x === 0 && y === 0) {
```

Comment:
When will this happen?

Reply:
e.g. in 2d mode, when `e.movementX` and `e.movementY` are both evaluated as negative , and mouse x move is 0: https://github.com/cruise-automation/webviz/blob/master/packages/regl-worldview/src/camera/CameraListener.js#L157